presto-spi/src/main/java/io/prestosql/spi/security/ConnectorIdentity.java

```diff
@@ -47,32 +47,13 @@ public String getUser()
         return role;
     }
 
-    @Override
-    public boolean equals(Object o)
-    {
-        if (this == o) {
-            return true;
-        }
-        if (o == null || getClass() != o.getClass()) {
-            return false;
-        }
-        ConnectorIdentity that = (ConnectorIdentity) o;
-        return Objects.equals(user, that.user) &&
-                Objects.equals(principal, that.principal);
-    }
-
-    @Override
-    public int hashCode()
-    {
-        return Objects.hash(user, principal);
-    }
-
     @Override
     public String toString()
     {
         StringBuilder sb = new StringBuilder("ConnectorIdentity{");
         sb.append("user='").append(user).append('\'');
         principal.ifPresent(principal -> sb.append(", principal=").append(principal));
+        role.ifPresent(role -> sb.append(", role=").append(role));
```

Comment:
Separate this commit into:
- `Remove ConnectorIdentity#equals`
- `Improve ConnectorIdentity#toString`

`toString` is a minor change, but change to `equals` is not and should not be hidden in a innocently-looking "refactor ..." commit